### PR TITLE
Working

### PR DESCRIPTION
**Pull request by Hamas2000**
This is my third project. I created a tribute page, and I used html and css in my project.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+body {
+  background-color: rgb(227, 219, 212);
+}
+
+#img-caption {
+  text-align: center;
+  font-size: smaller;
+}
```
Comment (ahzamir) [inline style.css:33]: - Your img element should have a display of block.
- Failed:Your #image should have a max-width of 100%.
- Failed:Your #image should be centered within its parent.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ </p>
+ <br>
+ <p id="tribute-info">In 2016, Suleiman founded the Muslim think tank Yaqeen Institute for Islamic Research. He is also the founding director of MUHSEN (Muslims Understanding and Helping Special Education Needs), a nonprofit umbrella organization that aims to create more inclusive Muslim communities that better cater to the disabled and their families.</p>
+ <br>
+<p id="tribute-info">He was (as of May 2019) Adjunct Professor of Islamic Studies at Southern Methodist University, where he was also a member of the Maguire Ethics Center Advisory Board.</p>
+<br>
+<p id="tribute-info">He is (as of September 2022) Resident Scholar of the Valley Ranch Islamic Center and co-chair of Faith Forward Dallas at Thanks-Giving Square.</p>
+<p id="extra">If you have time, you should read more about this incredible human being on his <a id="tribute-link" target="_blank" rel="noopener" href="https://www.facebook.com/imamomarsuleiman/">Facebook</a>.</p>
+</main>
+</body>
```
Comment (ahzamir) [inline index.html]: Your codes are not alligned correctly and are not clean try to make their spacing right.
- [**Suggession**] - On your HTML file, at the bottom right side of your vscode click on `Spaces`  then click on `Index using spaces`, and then select `2` from there. Then align your components according to the 2 spaces.
- Example:
```
<div>
  <p>...</p>
  <h1>.... </h1>
  <img ... />
</div>
```

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Hamas2000 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Copy your HTML & CSS file codes and paste them to the freecodecamp website on the Tribute Page project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.
- Although you added some basic design according to the requirements, I suggest you add some more design to your elements and page so it looks more cool and you can share it as a proud moment on your social media.
- **Check the inline comments below.**

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Tribute Page project task and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html (hunk 2 above)
Comment (Hamas2000) [reply index.html]: @ahzamir Thank you for your comment. I will go further and add some more styles to the page.

**Comment by Hamas2000** [comment]
@ahzamir Please check my project now. I brought some changes, and while testing on freecodecamp, name css file to style.css.

**Review by ahzamir — CHANGES_REQUESTED** [review]
@Hamas2000 

- Althought you have solved the requirement issue and now your code passes all checks on freecodecamp website but still your code is not clean:
- And for cleaning you code:
  - At the bottom right side of your vscode click on `Spaces` then click on `Index using spaces` and choose 2 from there.
  - Then according to the lines on the screen all child elements should be 2 spaces or one line ahead of their parent element and the closing tag should be in the same line as the opening tag!
  
  - Your body spacing is not aligned correctly:
  - All child elements should be two spaces ahead of their parent element.

**Example:**
```
<div>
  <h1> ... </h1>
  <p>
  .
  .
  </p>
  <div>
    <img ... />
    more elements ...
  </div>
</div>
```

**Comment by Hamas2000** [comment]
@ahzamir I am done, review it please. Thank you in advance.

**Review by ahzamir — APPROVED** [review]
### Hi @Hamas2000 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Tribute Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**